internal/build/buildkit_printer.go

```diff
@@ -10,84 +10,112 @@ import (
 
 type buildkitPrinter struct {
 	output io.Writer
-	vMap   map[digest.Digest]*vertexAndLogs
+	vData  map[digest.Digest]*vertexAndLogs
+	vOrder []digest.Digest
```

Comment:
This is a space-for-time tradeoff where we keep a separate list of the digests, in order so that we can print them in order.